govcd/nsxt_application_profile.go

```diff
@@ -78,6 +78,35 @@ func (org *Org) GetNsxtAppPortProfileByName(name, scope string) (*NsxtAppPortPro
 	return getNsxtAppPortProfileByName(org.client, name, queryParameters)
 }
 
+// GetNsxtAppPortProfileByName allows users to retrieve Application Port Profiles for specific scope.
+// More details in documentation for types.NsxtAppPortProfile
+//
+// Note. Names are enforced to be unique per scope
+func (vdc *Vdc) GetNsxtAppPortProfileByName(name, scope string) (*NsxtAppPortProfile, error) {
+	queryParameters := copyOrNewUrlValues(nil)
```

Comment:
not sure need it, but looks like we could have a private function which accepts only ID which is later added to _context  :)

Reply:
I'll probably refrain from it, because this function already calls a private function `getNsxtAppPortProfileByName` which would make it a "2 step" private function call